shared/reducers/wallets.js

```diff
@@ -336,11 +336,14 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
       })
     case WalletsGen.walletDisclaimerReceived:
       return state.merge({acceptedDisclaimer: action.payload.accepted})
-    // Saga only actions
     case WalletsGen.acceptDisclaimer:
       return state.merge({
         acceptingDisclaimerDelay: true,
       })
+    case WalletsGen.resetAcceptingDisclaimer:
+      return state.merge({
+        acceptingDisclaimerDelay: false,
```

Comment:
Should you also reset `acceptedDisclaimer`?

Reply:
Hm I don't think so, `acceptedDisclaimer` can only be true if the service says so, and then `wallets-and-details/container` will redirect to the main wallets tab